patterns/stateful/README.md

```diff
@@ -40,13 +40,19 @@ In addition, the following properties are configured on the nodegroup volumes:
 
 See [here](https://aws-ia.github.io/terraform-aws-eks-blueprints/getting-started/#prerequisites) for the prerequisites and steps to deploy this pattern.
 
-## Validate
+## Validate Storage
 
-For validating `velero` see [here](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/modules/kubernetes-addons/velero#validate)
+For validating `velero` see [here](https://aws-ia.github.io/terraform-aws-eks-blueprints/tree/main/patterns/stateful/README.md#Validate-Velero-Install)
```

Comment:
since this is self referencing and you've copied the validation below (thank you!) we can remove this line.